benchmark/single-source/StringRepeating.swift

```diff
@@ -0,0 +1,83 @@
+//===--- StringRepeating.swift -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+
+public let benchmarks = [
+  BenchmarkInfo(name: "String.initRepeating.1AsciiChar.Count100",
+                runFunction: run_singleAsciiCharacterCount100,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.26AsciiChar.Count2",
+                runFunction: run_26AsciiCharactersCount2,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.33CyrillicChar.Count2",
+                runFunction: run_33CyrillicCharactersCount2,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.longMixedString.Count100",
+                runFunction: run_longMixedStringCount100,
+                tags: [.validation, .api, .String])
+]
+
+@inline(never)
+func repeating(_ i: String, count: Int) -> String {
+  let s = String(repeating: getString(i), count: count)
```

Comment:
What is the purpose of this function? Why must it never be inlined?

Reply:
Many, if not all benchmarks for `String` follow the sma pattern. Take, e.g. `benchmark/single-source/StringBuilder.swift`. Here I just follow pattern.
As I understand, `@inline(never)` is needed here to forbid some optimizations.

Reply:
I'm not seeing any examples in the file you've referred to where a `String` API is wrapped in a function like this. Can you link to the exact line you're thinking of?